[DOC-706] xCluster limitations and CDC (#28143)
yugabyte/yugabyte-db: docs/content/preview/deploy/multi-dc/async-replication/async-transactional-failover.md

Use the [nextval](../../../../api/ysql/exprs/sequence_functions/func_nextval/) function to set the sequence next values appropriately.

### Fix CDC

If you are using CDC to move data out of YugabyteDB, note that failover may incur data loss for your CDC replication; data lost on the CDC target may be different from data lost on the xCluster target.

You can fix CDC in either of the following ways:

> - Start CDC on B (that is, create publications and slots on B). This resumes CDC where you left off (subject to possible data loss).

Reviewer: could CDC person check this?  I am dubious.

xCluster can rewind backwards from where the source universe was quite a bit.  That implies we would be sending the same data twice to CDC if we just start CDC from where it "left off"

I'm also confused how CDC knows where it left off on the source given that I do not believe we replicate the CDC state table

> - Start CDC on B (that is, create publications and slots on B). This resumes CDC where you left off (subject to possible data loss).

Reviewer 2: ```suggestion
- Start CDC on B (that is, create publications and slots on B). This resumes CDC from the failover point (subject to possible data loss).
```
